core/config/project_settings.cpp

```diff
@@ -1467,6 +1467,8 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("display/window/size/transparent", false);
 	GLOBAL_DEF("display/window/size/extend_to_title", false);
 	GLOBAL_DEF("display/window/size/no_focus", false);
+	GLOBAL_DEF("display/window/hdr/enabled", false);
+	GLOBAL_DEF("display/window/hdr/max_luminance", 0.0f);
```

Comment:
Add a property hint to disallow negative values:

```suggestion
	GLOBAL_DEF(PropertyInfo(Variant::FLOAT, "display/window/hdr/max_luminance", PROPERTY_HINT_RANGE, "0,1500,1,or_greater"), 0.0f);
```

The slider's upper range is set to 1500 nits, which is roughly what today's best consumer-grade displays can show in practice in a small window. You can still manually set a higher value by typing it in the project settings, which can be useful when testing on professional reference HDR displays.

Reply:
Applied, thanks.